dev_requirements.txt

```diff
@@ -7,4 +7,5 @@ pytest
 ptvsd
 mypy
 black
-types-PyYAML
\ No newline at end of file
+types-PyYAML
+docutils==0.18.1
```

Comment:
I think someone opened an issue recently about `docutils`, I believe it's a dep of one of our deps. Are you seeing `docutils` issues with the latest release? If so, we should pin it like we do with `mistune` (a dep of `m2r`( https://github.com/Azure/autorest.python/blob/autorestv3/setup.py#L51

Reply:
Moving to https://github.com/Azure/autorest.python/pull/1351.

Pinning the `docutils` version in setup.py so that it don't need to be pinned in both requirements.txt and dev_requirements.txt